tests/user/user_log_samples/alter_user_additional_secret_added_error.log

```diff
@@ -271,9 +271,9 @@
           "reasonCode": 8,
           "image": "ALTUSER SQUIDWRD  NOSPECIAL      OMVS     (HOME        ('/u/clarinet') NOPROGRAM      UID         (********))",
           "messages": [
-            "IKJ56702I INVALID UID, 90000000000",
-            "IKJ56701I MISSING OMVS UID+",
-            "IKJ56701I MISSING OMVS USER ID (UID), 1-10 NUMERIC DIGITS"
+            "REDACTED MESSAGE CONCERNING UID, REVIEW DOCUMENTATION OF IKJ56702I FOR MORE INFORMATION",
+            "REDACTED MESSAGE CONCERNING UID, REVIEW DOCUMENTATION OF IKJ56701I FOR MORE INFORMATION",
+            "REDACTED MESSAGE CONCERNING UID, REVIEW DOCUMENTATION OF IKJ56701I FOR MORE INFORMATION"
           ]
         }
       ]
```

Comment:
Does this still belong in user admin anymore? Should all secret redaction be tested in common since it is a common feature?

Reply:
Yes, my mistake, I forgot we did testing in user debug logging. I have moved this to common and will add comparable debug_logging testing for the other function groups as well.